src/coreComponents/mesh/generators/Region.hpp

```diff
@@ -0,0 +1,76 @@
+
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2016-2024 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2024 Total, S.A
+ * Copyright (c) 2018-2024 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2023-2024 Chevron
+ * Copyright (c) 2019-     GEOS/GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+/**
+ * @file Region.hpp
+ */
+
+#ifndef GEOS_MESH_GENERATORS_REGION_HPP
+#define GEOS_MESH_GENERATORS_REGION_HPP
+
+#include "MeshComponentBase.hpp"
+
+namespace geos
+{
+
+/**
+ * @brief Region parameters with Group capabilities
+ *
+ * This class has dataRepository::Group capabilities to allow for XML input.
+ *
+ */
+class Region : public MeshComponentBase
```

Comment:
@rrsettgast @CusiniM is ok to call it Region?

Reply:
Region seems like it is too generic? How does this "Region" differ from the other "Region"?